taichi/codegen/llvm/llvm_codegen_utils.cpp

```diff
@@ -31,7 +74,17 @@ void check_func_call_signature(llvm::Value *func,
   for (int i = 0; i < num_params; i++) {
     auto required = func_type->getFunctionParamType(i);
     auto provided = arglist[i]->getType();
+    /*
```

Comment:
btw, is there a way to "link" the conflicting types imported from files instead of renaming them?

Reply:
The linker treats same types with different names as one, so the different type names do not confuse the linker. Therefore this may not be a problem after the module splitting.